postal/dependency_mappings.go

```diff
@@ -0,0 +1,53 @@
+package postal
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type DependencyMappingResolver struct{}
+
+func NewDependencyMappingResolver() DependencyMappingResolver {
+	return DependencyMappingResolver{}
+}
+
+// Reference file structure for bindings directory
+// - bindings
+//    - some-binding
+//       - type -> dependency-mapping
+// 			 - some-sha -> some-uri
+//       - other-sha -> other-uri
+
+// Given a target dependency, look up if there is a matching dependency mapping at the given binding path
+func (d DependencyMappingResolver) FindDependencyMapping(sha256, bindingPath string) (string, error) {
+	allBindings, err := filepath.Glob(filepath.Join(bindingPath, "*"))
+	if err != nil {
+		return "", err
+	}
+
+	for _, binding := range allBindings {
+		bindType, err := ioutil.ReadFile(filepath.Join(binding, "type"))
+		if err != nil {
+			return "", fmt.Errorf("couldn't read binding type: %w", err)
+		}
+
+		if strings.Contains(string(bindType), "dependency-mapping") {
```

Comment:
```suggestion
		if strings.TrimSpace(string(bindType)) == "dependency-mapping" {
```

This should probably be a bit more restrictive so that we don't get a false match with a type like `dependency-mapping-something`.